File: February 24, 2026.py
from datetime import datetime, timedelta

def count_business_days(start, end):

    count = 0
    increment_day = datetime.strptime(start, "%Y-%m-%d").date()
    end_day = datetime.strptime(end, "%Y-%m-%d").date()
    
    
    while increment_day <= end_day:
        if increment_day.weekday() < 5:
            count += 1
        increment_day += timedelta(days=1)

    return count

File: February 24, 2026.js
function countBusinessDays(start, end) {

  let count = 0;
  let incrementDate = new Date(start + "T00:00:00");
  let endDate = new Date(end + "T00:00:00"); 


  while (incrementDate <= endDate) {

    const day = incrementDate.getDay()
    if((day !== 0) && (day!==6)) {
      count++;
    }

    incrementDate.setDate(incrementDate.getDate() + 1)
  }
  
  return count;
}